### PR TITLE
#166492273 Read Articles

### PR DESCRIPTION
**Pull request by mwinel**
#### What does this PR do?
This PR adds the read articles feature. A user can either signup or login and read a particular article. The user will be welcomed with a landing page that has a list of articles. When they click on one, they are then redirected to the article page to read the article.

#### Description of Task to be completed
- Retrieve all articles

#### How should this be manually tested?
1. clone the repo
2. run `npm install`
3. run `npm start`
4. login on `http://localhost:3000/login`
5. see all articles here `http://localhost:3000/`
6. read an article using this link `http://localhost:3000/articles/<article-id>/`

#### Any background context you want to provide?
The HTML templating and CSS styling is not yet done.

#### What are the relevant pivotal tracker stories?
[#166492273](https://www.pivotaltracker.com/story/show/166492273)

#### Screenshots (if appropriate)
Screen Shot 2019-06-16 at 14 15 42

### CODE REVIEW

**Comment by njeri-ngigi** [comment]
please add a screenshot of your work

File: src/Router.js [hunk 1]
```diff
@@ -25,6 +27,9 @@ class Router extends Component {
         <Route path="/articles/create" component={ArticleContainer} />
         <Route path="/profile" component={ProfileComponent} />
         <Route path="/editprofile" component={editProfileComponent} />
+        <Route path="/editprofile" component={editProfileComponent} />
```
Comment (njeri-ngigi) [inline src/Router.js]: this route seems repeated

**Review by KabohaJeanMark — COMMENTED** [review]
Good job on adding the articles link to the navigation bar on the home page. This makes it easier to access.

File: src/app/components/articles/getArticleComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Navbar from '../../components/Navbar';
+
+const ArticleComponent = (props) => {
+  const { article: { title, body, author, description } } = props;
```
Comment (KabohaJeanMark) [inline src/app/components/articles/getArticleComponent.jsx:5]: Good work on using destructuring here

**Review by KabohaJeanMark — APPROVED** [review]
Good job Nelson. This has worked out on my machine so far. We can get all the articles and get one article by hitting both the endpoints.

**Review by njeri-ngigi — APPROVED** [review]
good job @mwinel

**Pull request merged**